classes/session/SessionManager.php

```diff
@@ -39,13 +39,16 @@ public function __construct(SessionDAO $sessionDao, PKPRequest $request)
         $this->sessionDao = $sessionDao;
 
         // Configure PHP session parameters
-        ini_set('session.use_trans_sid', 0);
-        ini_set('session.serialize_handler', 'php');
-        ini_set('session.use_cookies', 1);
-        ini_set('session.name', Config::getVar('general', 'session_cookie_name')); // Cookie name
+        ini_set('session.name', Config::getVar('general', 'session_cookie_name'));
         ini_set('session.cookie_lifetime', 0);
         ini_set('session.cookie_path', Config::getVar('general', 'session_cookie_path', $request->getBasePath() . '/'));
         ini_set('session.cookie_domain', $request->getServerHost(null, false));
```

Comment:
Some lines below, there's some code updating the `session.cookie_domain`, I think it makes sense to bring it before the `session_start()` 🤔

Reply:
@asmecher I tried to find this issue in GitHub and failed, thanks for the link!

The discussion makes sense, but the problem still exists (Chrome + OJS 3.3.0):
1. Accessed `journal.xyz`
   > Got a cookie with the domain `.journal.xyz`.
2. Accessed `www.journal.xyz`
   > The previous cookie is visible at the subdomain, and no extra cookie was created.
3. Tried to login and got redirected back to the login form
   > A new cookie with the domain `.www.journal.xyz` was created, but the previous cookie wasn't removed, because:
    - The code above only drops domainless cookies
    - The previous cookie failed to be re-used, because this line doesn't work (it's not possible to set these session constants when there's an active session): https://github.com/pkp/pkp-lib/blob/ef79d952408a9c91fa0b38cfbdb407df69429704/classes/session/SessionManager.php#L78

FYI, the login form generated these `Set-Cookie` headers at the response:
```http
Set-Cookie: OJSSID=mlamfo2l6deevkhnjmflsmk1tk; path=/; domain=www.journal.xyz
Set-Cookie: OJSSID=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; Max-Age=0; path=/
Set-Cookie: OJSSID=mlamfo2l6deevkhnjmflsmk1tk; path=/; domain=www.journal.xyz; HttpOnly
Set-Cookie: OJSSID=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; Max-Age=0; path=/
Set-Cookie: OJSSID=mlamfo2l6deevkhnjmflsmk1tk; expires=Thu, 09-Feb-2023 19:43:34 GMT; Max-Age=2592000; path=/; domain=www.journal.xyz; HttpOnly
```

I'll check the other bugzilla link and update the PR.